### PR TITLE
US132794 - Quizzing LHP Skeleton

### PR DESCRIPTION
**Pull request by maddiegabriel**
[US132794 - [Consistent Evaluation][Quizzing][UI] Implement Loading Skeleton for LHP](https://rally1.rallydev.com/#/42960320374d/custom/236803223728?detail=%2Fuserstory%2F611572657323
)

[Design (Attempt Page)](https://xd.adobe.com/view/9007b2d1-d470-499c-a206-97f9a915ab50-c554/screen/868d4287-fd36-4028-98fb-65609d291777/specs/)
SKELETON_LHP_ATTEMPT

[Design (Completion Summary)](https://xd.adobe.com/view/9007b2d1-d470-499c-a206-97f9a915ab50-c554/screen/1f478597-a064-4e27-a5ba-97ac83681fe9/specs/)
[GIF coming]

### CODE REVIEW

File: components/left-panel/discussions/consistent-evaluation-discussion-post-page.js [hunk 1]
```diff
@@ -193,14 +193,15 @@ export class ConsistentEvaluationDiscussionPostPage extends SkeletonMixin(RtlMix
 				${this._renderDiscussionRatingSkeleton()}
 				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-post-title-skeleton"></div>
 				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-post-date-skeleton"></div>
-				<div class="d2l-skeletize d2l-skeletize d2l-consistent-evaluation-discussion-post-text-skeleton"></div>
-				<div class="d2l-skeletize d2l-skeletize d2l-consistent-evaluation-discussion-post-text-skeleton"></div>
-				<div class="d2l-skeletize d2l-skeletize d2l-consistent-evaluation-discussion-post-last-text-skeleton"></div>
+				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-post-text-skeleton"></div>
+				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-post-text-skeleton"></div>
+				<div class="d2l-skeletize d2l-consistent-evaluation-discussion-post-last-text-skeleton"></div>
```
Comment (maddiegabriel) [inline components/left-panel/discussions/consistent-evaluation-discussion-post-page.js:198]: I just noticed these were doubled

**Comment by Scy-D2L** [comment]
The gifs are cool to functionally demo but it might be a good idea to have a static screenshot of the skeleton (so we can see it for more than 0.1 seconds to verify it matches the design ;)

**Comment by maddiegabriel** [comment]
> The gifs are cool to functionally demo but it might be a good idea to have a static screenshot of the skeleton (so we can see it for more than 0.1 seconds to verify it matches the design ;)

absolutely 😆 I'll have time to come back to this and fix it up in the afternoon, I'll ping you when it's ready to review

**Comment by tdevos-d2l** [comment]
Hehe sorry about the merge conflicts I added

**Pull request closed without merging**